packages/cozy-client/src/CozyClient.spec.js

```diff
@@ -1402,6 +1401,42 @@ describe('CozyClient', () => {
         receiveQueryResult('allTodos', fakeResponse)
       )
     })
+
+    const setupOnError = () => {
+      const error = new TypeError('Failed to fetch')
+      requestHandler.mockReturnValueOnce(Promise.reject(error))
+      const onError = jest.fn().mockReturnValueOnce(() => true)
+      return onError
+    }
+
+    it('should call onError callback when an error is catched. It is passed as argument', async () => {
```

Comment:
should call onError callback when an error is thrown. Callback passed as argument